### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by sagararyal** [Plant-for-the-Planet/planet-webapp#174]
- [x] Alt Text of decorative image should be blank.

- [x] Navbar should not have role button but only be a link.

- [x] Planet logo should have alt text (About Plant-for-the-Planet)

**Comment by sarveshpro** [comment]
also added alt text to all non-decorative images
